CHANGELOG.md

```diff
@@ -1,9 +1,40 @@
-# Changelog
+# Change Log
```

Comment:
I think it is better to keep it as `Changelog`.

https://keepachangelog.com/en/1.1.0/